Documentation/api.md

```diff
@@ -122,6 +122,7 @@ AlertmanagerSpec is a specification of the desired behavior of the Alertmanager
 | logLevel | Log level for Alertmanager to be configured with. | string | false |
 | replicas | Size is the expected size of the alertmanager cluster. The controller will eventually make the size of the running cluster equal to the expected size. | *int32 | false |
 | retention | Time duration Alertmanager shall retain data for. Default is '120h', and must match the regular expression `[0-9]+(ms\|s\|m\|h)` (milliseconds seconds minutes hours). | string | false |
+| maxConcurrency | Number of concurrent queries that can be run at once. | *int32 | false |
```

Comment:
As there are multiple query flag, how about moving this into a separate `query` configuration object?

Reply:
Sure，will do

Reply:
I've added a seperate query config object, would you help to review?
Thanks
Ben

Reply:
It seems that jsonnetfile.lock.json always try to get code from a certain release, which causes the CI test broken because of the old 0prometheus-operator-0prometheusCustomResourceDefinition.yaml

Reply:
Yes the generated CRDs in kube-prometheus will only be adapted on a release of the prometheus-operator, they can stay as they are on master currently within this PR.

Reply:
But how to handle the CI error?